docs/user_manual/calculations.md

```diff
@@ -57,10 +57,10 @@ Output:
 - Power flow through branches
 - Deviation between measurement values and estimated state
 
-In order to perform a state estimation, the system should be observable. If the system is not observable, the calculation will 
-raise a sparse matrix error; the matrix reprensentation of the system is too sparse to solve and particular so when it is
-singular. In short, meeting the requirement of observability indicates that the system is either an overdetermined system (when the number of measurements is larger than the number of
-unknowns) or a balanced system (when the number of measurements is equal to the number of unknowns). For each node, there are two unknowns, `u` and `u_angle`, so the following conditions should be met:
+In order to perform a state estimation, the system should be observable. If the system is not observable, the calculation will either raise a not observable error or 
+a sparse matrix error.
```

Comment:
we most likely should update the default error message of the sparse matrix error (also check whether the python regex needs to be updated). it's not up to date anymore after this change.

Reply:
the word `usually` may be misleading and/or confusing, though...

```
"Sparse matrix error, possibly singular matrix!\n"
If you get this error from state estimation, it usually means the system is not fully observable, i.e. not enough measurements.
```

Reply:
Changed error message.